Features1/tests/features/editContact.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Edit Contacts
+    As a user
+    I want to edit my existing contacts 
+    So that i can make some update and display it in my contact list
+
+    Background:
+        Given a user is navigated to the contact page 
+        And has following contact list 
+            |name|email        |phone   |contact-type|
+            |abc |abc@gmail.com|98000000|Personal    |
+            |aaa |aaa@gmail.com|98111111|Professional|
+        And clicks on the "Edit" button , data is displayed on the edit contact form
```

Comment:
you have created 2 users so 2 edit button is there which edit button would you click?